### DISCUSSION TITLE
Arduino Mega2560: which port (+index) represents which pin?

### DISCUSSION THREAD

**Discussion by stllvdv** [shanteacontrols/OpenDeck#217] [Q&A] [answered]
Hello @paradajz et al.,

I started building things around the Raspberry Pi Pico, but want to finish something with the Arduino Mega2560 as I can't return it. I want to add multiplexers, but I'm a bit confused by the port-naming scheme. [The pinouts](https://github.com/shanteacontrols/OpenDeck/wiki/Arduino-Mega2560) help tremendously, but how would I know within the YML-file which pin e.g. `port: "B" index: 1` is referring to?

I apologize if this is yet another question with a ridiculously simple answer.

Thank you!

EDIT: Currently struggling with which ports and indexes I want to define for `s0, s1, s2, s3`

**Comment by paradajz** [comment, marked as answer]
Here's a PDF from Arduino website:

https://content.arduino.cc/assets/Pinout-Mega2560rev3_latest.pdf

**Reply by stllvdv** [reply]
Thanks!
